modules/api_docs/test/TestPlan.md

```diff
@@ -0,0 +1,53 @@
+# API Docs module Test Plan
+
+##  Overview
+
+The API docs modules takes the api descriptor of LORIS's modules and creates an 
+actionnable html page. Each endpoint of the selected schema is described and 
+can be tried providing the required parameters. It uses 
+[Swagger-ui](https://github.com/swagger-api/swagger-ui) to create the webpage.
+
+##  Features
```

Comment:
Why is this in the test plan and not the README?

Reply:
The is no other reason than providing a summary of the features to test to the tester.
It can be added to the README if you want. Or it can be remove from this file. Or else...